monorail/tiles.py

```diff
@@ -309,6 +309,7 @@ def get_screen_position( self, length ):
 
 
         if in_dir == out_dir.get_opposite():
+            # TODO: not sure what here

```

Comment:
I have doubts it is good idea to put this into source code. Maybe you should create an issue instead?